File: Most-Recursion/Questions/Subsequences.py
def sub(s,i,curr):
    if(i == len(s)):
        print(f'python method print {curr}')
        return
    sub(s,i+1,curr+s[i])
    sub(s,i+1,curr)
indx = 0
s = 'abc'
curr = ''
sub(s,indx,curr)

File: Most-Recursion/Questions/Subsequences.js
function subsequences(s,i,curr){
    if(i == s.length){
        console.log(`javaScript method print ${curr}`);
        return;
    }
    subsequences(s,i+1,curr+s[i]); // Take
    subsequences(s,i+1,curr); // Leave
}
let s = 'abc';
let curr = '';
let index = 0;
subsequences(s,index,curr);